meja/envi.ml

```diff
@@ -0,0 +1,124 @@
+open Core_kernel
```

Comment:
nit: I prefer `Env` to `Envi` everywhere, not sure if this is a preference others share

Reply:
We get a module conflict with OCaml's compiler libs if we do this :cry: